### DISCUSSION TITLE
Connecting display

### DISCUSSION THREAD

**Discussion by souravlal** [marcel-licence/esp32_midi_sampler#42] [Q&A] [answered]
@marcel-licence Do I have to do the modifications on the ESP32 Audio kit( removing and replacing resistors) inorder for the display to work?

**Comment by marcel-licence** [comment, marked as answer]
Yes. I think I also mentioned it in the video that it is required because there are capacitors parallel to the IOs. So the removal is required. You don't need to place the new resistors.
